src/components/region-common.js

```diff
@@ -14,6 +14,7 @@ import * as Utils from 'src/libs/utils';
 const unknownRegionFlag = '❓';
 export const getRegionInfo = (location, locationType) => {
   const regionDescription =
+    // eslint-disable-next-line no-nested-ternary
     locationType === locationTypes.multiRegion ? `${location} (${locationTypes.multiRegion})` : location ? location.toLowerCase() : 'UNKNOWN';
```

Comment:
Interesting that this uses `location` in the first branch but checks for its presence in the other.